### PR TITLE
Japanese docs

### LINKED ISSUE #5285: add japanese translation for docs translation
**Is your feature request related to a problem? Please describe.**
Add translation of documentation

**Describe the solution you'd like**
Add translation in japanese

### PR DESCRIPTION
**Pull request by isamu-isozaki**
# What does this PR do?
Adds Japanese documentations

Fixes #5285

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/diffusers/blob/main/CONTRIBUTING.md)?
- [x] Did you read our [philosophy doc](https://github.com/huggingface/diffusers/blob/main/PHILOSOPHY.md) (important for complex PRs)?
- [x] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link to it if that's the case.
- [x] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/diffusers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.
Since, it's the docs, also @stevhliu and @yiyixuxu

### CODE REVIEW

**Review by stevhliu — COMMENTED** [review]
Nice work! 👏

You'll also need to modify the `build_documentation.yml` and `build_pr_documentation.yml` file to include the language code for Japanese (`ja`) to build the docs.

**Comment by isamu-isozaki** [comment]
@stevhliu Thanks for the review! And should be fixed

File: .github/workflows/build_documentation.yml [hunk 1]
```diff
@@ -16,7 +16,7 @@ jobs:
       install_libgl1: true
       package: diffusers
       notebook_folder: diffusers_doc
-      languages: en ko zh
+      languages: en ko zh jp
```
Comment (stevhliu) [inline .github/workflows/build_documentation.yml:19]: The ISO 639-1 language code for Japanese is `ja`

```suggestion
      languages: en ko zh ja
```

File: .github/workflows/build_pr_documentation.yml [hunk 2]
```diff
@@ -15,4 +15,4 @@ jobs:
       pr_number: ${{ github.event.number }}
       install_libgl1: true
       package: diffusers
-      languages: en ko zh
+      languages: en ko zh jp
```
Comment (stevhliu) [inline .github/workflows/build_pr_documentation.yml:18]:
```suggestion
      languages: en ko zh ja
```

**Review by stevhliu — APPROVED** [review]
Thanks for the changes, just two minor ones remaining :)

File: .github/workflows/build_pr_documentation.yml (hunk 2 above)
Comment (isamu-isozaki) [reply .github/workflows/build_pr_documentation.yml]: Nice catch! And I fixed the folder name to be ja too

**Comment by stevhliu** [comment]
Thanks again for your contribution! 🤗

**Pull request merged**